sequencer/api/config.go

```diff
@@ -6,19 +6,19 @@ import (
 )
 
 type rollupConstants struct {
-	PublicConstants         common.RollupConstants `json:"publicConstants"`
-	MaxFeeIdxCoordinator    int                    `json:"maxFeeIdxCoordinator"`
-	ReservedIdx             int                    `json:"reservedIdx"`
-	ExitIdx                 int                    `json:"exitIdx"`
-	LimitDepositAmount      *big.Int               `json:"limitDepositAmount"`
-	LimitL2TransferAmount   *big.Int               `json:"limitL2TransferAmount"`
-	LimitTokens             int                    `json:"limitTokens"`
-	L1CoordinatorTotalBytes int                    `json:"l1CoordinatorTotalBytes"`
-	L1UserTotalBytes        int                    `json:"l1UserTotalBytes"`
-	MaxL1UserTx             int                    `json:"maxL1UserTx"`
-	MaxL1Tx                 int                    `json:"maxL1Tx"`
-	InputSHAConstantBytes   int                    `json:"inputSHAConstantBytes"`
-	NumBuckets              int                    `json:"numBuckets"`
+	PublicConstants common.RollupConstants `json:"publicConstants"`
+	// MaxFeeIdxCoordinator    int                    `json:"maxFeeIdxCoordinator"`
+	ReservedIdx             int      `json:"reservedIdx"`
+	ExitIdx                 int      `json:"exitIdx"`
+	LimitDepositAmount      *big.Int `json:"limitDepositAmount"`
+	LimitL2TransferAmount   *big.Int `json:"limitL2TransferAmount"`
```

Comment:
`LimitL2TransferAmount` , `LimitTokens`,  `L1CoordinatorTotalBytes`,  `NumBuckets`, `ExchangeMultiplier` could be removed.